exir/sym_util.py

```diff
@@ -41,17 +41,23 @@ def eval_upper_bound(maybe_symint: Union[int, torch.SymInt]) -> Optional[int]:
     expr = node.expr
     var_range: ValueRanges = bound_sympy(expr, shape_env.var_to_range)
     upper_bound = var_range.upper
+
```

Comment:
add a comment that this could be deleted when we move forward the pin everywhere